include/controller.h

```diff
@@ -53,6 +53,8 @@ class controller {
 		return refresh_on_start;
 	}
 	void enqueue_url(const std::string& url,
+        const std::string& title,
+        const time_t pubDate,
```

Comment:
Please use tabs for indentation. You can run `make fmt` and then `git add --patch` the changes to the code you added.